macros/edr/tests/on_run_end/merge_tests_results.sql

```diff
@@ -2,91 +2,48 @@
     {% if execute and flags.WHICH == 'test' %}
         {% set database_name, schema_name = elementary.get_package_database_and_schema('elementary') %}
         {% set temp_metrics_tables, temp_anomalies_tables, temp_schema_changes_tables = elementary.get_temp_tables_from_graph(database_name, schema_name) %}
-        {{ elementary.merge_data_monitoring_metrics(database_name, schema_name, temp_metrics_tables) }}
-        {{ elementary.merge_data_monitoring_anomalies(database_name, schema_name, temp_anomalies_tables) }}
-        {{ elementary.merge_schema_changes_alerts(database_name, schema_name, temp_schema_changes_tables) }}
+        {{ elementary.merge_temp_tables_data_to_target(database_name, schema_name, temp_metrics_tables, 'metrics', 'id') }}
```

Comment:
1. Use the target ('data_monitoring_metrics' / 'alerts_data_monitoring' / 'alerts_schema_changes') as the third param (instead of defining a temp_data_type and then finding based on it the target, simply pass the target)